### ISSUE TITLE
Bug: error when recording an Express app remotely

### ISSUE THREAD

**Issue by ptrdvrk** [applandinc/appmap-agent-js#41]
This was a random bug when stopping recording that I could not reproduce again:

```
npx appmap-agent-js
APPMAP-WARNING command "git describe --abbrev=0" on cwd "/Users/ptrdvrk/work/github/js/freshlytics" failed with 128 >> fatal: No annotated tags can describe '75a167bafbc6af293ec02c107e1d704c30537e00'.
However, there were unannotated tags: try --tags.

APPMAP-WARNING command "git describe --long" on cwd "/Users/ptrdvrk/work/github/js/freshlytics" failed with 128 >> fatal: No annotated tags can describe '75a167bafbc6af293ec02c107e1d704c30537e00'.
However, there were unannotated tags: try --tags.

APPMAP-INFO Trace port: 63005
APPMAP-INFO Track port: 3003
APPMAP-INFO node ./dist ...
(node:31932) ExperimentalWarning: --experimental-loader is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
APPMAP-INFO Recorder 'remote' caught process 31932
APPMAP-INFO recording /Users/ptrdvrk/work/github/js/freshlytics/dist
APPMAP-INFO Received remote recording request: POST /5pdu280k/record
APPMAP-INFO Received remote recording request: DELETE /5pdu280k/record
APPMAP-INFO Received remote recording request: POST /5pdu280k/record
APPMAP-INFO Received remote recording request: DELETE /5pdu280k/record
(node:31923) UnhandledPromiseRejectionWarning: TypeError [ERR_INVALID_URL]: Invalid URL: APPMAP-MANUFACTURED-BEGIN-RESPONSE
    at onParseError (internal/url.js:279:9)
    at new URL (internal/url.js:355:5)
    at parseURL (file:///Users/ptrdvrk/work/github/js/freshlytics/node_modules/@appland/appmap-agent-js/components/trace/appmap/event/data.mjs:16:5)
    at compileCallData (file:///Users/ptrdvrk/work/github/js/freshlytics/node_modules/@appland/appmap-agent-js/components/trace/appmap/event/data.mjs:62:36)
    at compileEventTrace (file:///Users/ptrdvrk/work/github/js/freshlytics/node_modules/@appland/appmap-agent-js/components/trace/appmap/event/index.mjs:48:18)
    at compileTrace (file:///Users/ptrdvrk/work/github/js/freshlytics/node_modules/@appland/appmap-agent-js/components/trace/appmap/index.mjs:48:17)
    at sendBackend (file:///Users/ptrdvrk/work/github/js/freshlytics/node_modules/@appland/appmap-agent-js/components/backend/default/index.mjs:66:17)
    at file:///Users/ptrdvrk/work/github/js/freshlytics/node_modules/@appland/appmap-agent-js/components/receptor-http/http/index.mjs:118:15
    at IncomingMessage.<anonymous> (file:///Users/ptrdvrk/work/github/js/freshlytics/node_modules/@appland/appmap-agent-js/components/http/node-http/index.mjs:83:49)
    at IncomingMessage.emit (events.js:412:35)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:31923) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:31923) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
APPMAP-INFO Received remote recording request: DELETE /5pdu280k/record
APPMAP-INFO Received remote recording request: GET /5pdu280k/record
^CAPPMAP-INFO > Killed with: SIGKILL
```

**Comment by lachrist** [comment]
This should be solved in 6.3.2 by https://github.com/applandinc/appmap-agent-js/commit/95eb0b7210eae5e0e7dc28b5ddaa6eff805b0dbf

**Comment by ptrdvrk** [comment]
No longer reproducible.
